packages/contentful--react-apps-toolkit/src/useAutoResizer.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  DialogExtensionSDK,
+  FieldExtensionSDK,
+  SidebarExtensionSDK,
+  WindowAPI,
+} from '@contentful/app-sdk';
+import { useEffect } from 'react';
+import { useSDK } from './useSDK';
+
+/**
+ * Starts/stops the auto resizer when the component is mounted/unmounted.
+ * Must be wrapped by SDKProvider.
+ * Can only be used when app is rendered in field, sidebar or dialog locations.
```

Comment:
```suggestion
 * Can only be used when app is rendered in field, sidebar or dialog locations.
 * @param {Object} props - Options to be passed to auto resizer
 * @param {Boolean} props.absoluteElements - Defines if auto resizer should consider absolut elements
```

Reply:
Ah yeah good spot. For better semantic sense, I have renamed `props` to `params` and included these JSDoc annotations.